src/demo-app/a11y/progress-spinner/progress-spinner-a11y.html

```diff
@@ -0,0 +1,16 @@
+<section>
+  <h2>Loading indicator (Indeterminate progress spinner)</h2>
+  <md-spinner color="indeterminate" [strokeWidth]="1" aria-label="Loading"></md-spinner>
+  <md-spinner color="accent" aria-label="Loading"></md-spinner>
+</section>
+
+<section>
+  <h2>Portion of completion (Determinate progress spinner) </h2>
+  <md-progress-spinner [strokeWidth]="1" color="primary"
+                       [value]="portionValue" aria-label="Portion complete">
```

Comment:
Could you make the labels something like "Portion of pizza eaten", "Portion of cake eaten"? (e.g. more "real life" examples)